package.json

```diff
@@ -73,10 +72,10 @@
         "release:latest": "DIST=latest BRANCH=master ./scripts/release.sh",
         "release:next": "DIST=next BRANCH=next ./scripts/release.sh",
         "release:cdn": "yarn setup; node ./scripts/prod.js",
-        "setup": "yarn install --frozen-lockfile; npm-run-all clean build:i18n",
+        "setup": "yarn install --frozen-lockfile --ignore-optional; npm-run-all clean build:i18n",
         "start": "npm-run-all setup start:examples",
         "start:dev": "yarn setup; LANGUAGE=en-US BABEL_ENV=development NODE_ENV=development node --max_old_space_size=8192 node_modules/webpack-dev-server/bin/webpack-dev-server.js --config scripts/webpack.config.js --mode development",
-        "start:examples": "EXAMPLES=true LANGUAGE=en-US REACT=true BABEL_ENV=development NODE_ENV=development node --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js server --config scripts/styleguide.config.js --mode development",
+        "start:examples": "EXAMPLES=true LANGUAGE=en-US REACT=true BABEL_ENV=development NODE_ENV=development node --openssl-legacy-provider --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js server --config scripts/styleguide.config.js --mode development",
```

Comment:
node 17 introduced some ssl security fixes: https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported

This option should not be kept long term. however, the impact is limited since we are only using a "server" in the context of local styleguidist